cromshell

```diff
@@ -127,48 +129,56 @@ function simpleUsage()
+  echo -e "   execution-status-count [workflow-id] [[workflow-id]...]           Get the summarized status of all jobs in the workflow."
+  echo -e "   timing [workflow-id] [[workflow-id]...]                           Open the timing diagram in a browser."
   echo -e
-  echo -e "  Logs"
-  echo -e "   logs [workflow-id] [[workflow-id]...]                             List the log files produced by a workflow"
-  echo -e "   fetch-logs [workflow-id] [[workflow-id]...]                       Download all logs produced by a workflow"
+  echo -e "  Logs:"
+  echo -e "   logs [workflow-id] [[workflow-id]...]                             List the log files produced by a workflow."
+  echo -e "   fetch-logs [workflow-id] [[workflow-id]...]                       Download all logs produced by a workflow."
   echo -e
-  echo -e "  Job Outputs"
-  echo -e "   list-outputs [workflow-id] [[workflow-id]...]                     List all output files produced by a workflow"
-  echo -e "   fetch-all [workflow-id] [[workflow-id]...]                        Download all output files produced by a workflow"
+  echo -e "  Job Outputs:"
+  echo -e "   list-outputs [workflow-id] [[workflow-id]...]                     List all output files produced by a workflow."
+  echo -e "   fetch-all [workflow-id] [[workflow-id]...]                        Download all output files produced by a workflow."
   echo -e
-  echo    "  Get email notification on job completion"
+  echo -e "  Get email notification on job completion:"
   echo -e "   notify [workflow-id] [daemon-server] email [cromwell-server]"
-  echo -e "        email          email address to which to send the notification"
-  echo -e "        daemon-server  server to run the notification daemon on"
-  echo
+  echo -e "        email             Email address to which to send the notification."
+  echo -e "        daemon-server     Server on which to run the notification daemon."
+  echo -e ""
   echo -e "  Display a list jobs submitted through cromshell:"
   echo -e "   list [-c] [-u]                                            "
-  echo -e "         -c    Color the output by completion status"
-  echo -e "         -u    Check completion status of all unfinished jobs"
-  echo -e
+  echo -e "         -c               Color the output by completion status."
+  echo -e "         -u               Check completion status of all unfinished jobs."
+  echo -e "" 
+  echo -e "  Clean up local cached list:"
+  echo -e "   cleanup [-s STATUS]    Remove completed jobs from local list."
+  echo -e "                          Will remove all jobs from the local list that are in a completed state,"
+  echo -e "                          where a completed state is one of: $(echo ${TERMINAL_STATES} | tr ' ' ',' )" 
+  echo -e "         -s STATUS        If provided, will only remove jobs with the given STATUS from the local list."
```

Comment:
should we have a -y flag which lets you skip the prompt and shoot yourself in the foot?

Reply:
For now, I'd say no.  I don't want to let the user be too reckless for now.